onnxruntime/core/framework/memcpy.cc

```diff
@@ -26,6 +26,7 @@ Status Memcpy::Compute(OpKernelContext* ctx) const {
                                                " X data:", X->DataRaw(), " Y data:", Y->DataRaw());
     }
   } else if (input_type_0->IsSparseTensorType()) {
+#if !defined(DISABLE_SPARSE_TENSORS)
     const auto* X = ctx->Input<SparseTensor>(0);
```

Comment:
May need to move this up a little so we hit the 'else' if sparse tensor support is disabled.